warble/functions.cc

```diff
@@ -69,10 +70,17 @@ bool Warble(Database* db, Any req, Any* rep) {
   } else {
     LOG(INFO) << "no parent id found " << parent_id;
   }
+  std::istringstream iss(text);
+  std::vector<std::string> result{std::istream_iterator<std::string>(iss), {}};
+  for (auto r : result) {
+    if (r[0] == '#') {
+      db->put("_hashtag_" + r, w_as_string);
```

Comment:
may be better to call a function here to notify the streaming requests that a Warble has been posted, instead of relying on polling in a while loop